flixel/FlxObject.hx

```diff
@@ -841,6 +846,11 @@ class FlxObject extends FlxBasic
 		return point.set(x + width * 0.5, y + height * 0.5);
 	}
 	
+	public inline function getHitbox():FlxRect
```

Comment:
I'd prefer to structure this code similar to ```getMidPoint``` (with optional parameter), That way developers can easily optimize code to avoid allocations or pooling costs.